daemon/endpoint.go

```diff
@@ -132,14 +134,32 @@ func NewPutEndpointIDHandler(d *Daemon) PutEndpointIDHandler {
 	return &putEndpointID{d: d}
 }
 
+func getEndpointAnnotator() endpoint.Annotator {
+	var annotator endpoint.Annotator
+	if k8s.IsEnabled() {
+		annotator = k8s.Client()
+	}
+	return annotator
+}
+
+func getHealthAnnotator() launch.Annotator {
+	var annotator launch.Annotator
+	if k8s.IsEnabled() {
+		annotator = k8s.Client()
```

Comment:
Don't need to create the variable you can directly return `nil` or `k8s.Client()`

Reply:
I'll remove these functions in the next upload.